### PR TITLE
Ash walker clothing additions & donor item

### PR DESCRIPTION
**Pull request by Detrender**
## About The Pull Request

I added some cosmetic clothes to ash walker vendors so as to allow for greater customization options.

## How This Contributes To The Skyrat Roleplay Experience

It gives some much needed cosmetic diversity to ash walkers in hopes of making them much more desirable as a ghost role option.
also a nifty cloak.
## Changelog
:cl:
add: added chestwrap and loincloth uniform
add: added added ash walker tunic
add: added ornate mantle
add: added ornate leg wraps
add: added mundane leg wraps
add: added ash walker waistcloth
add: added chestwrap and loincloth, ashwalker tunic, ornate mantle, ornate leg wraps, mantle, mundane leg wraps, ashwalker waistcloth, and loincloth to ashwalker vendor
add: added donator item "ashen cloak"
/:cl:
image
image
image
image
image
image

### CODE REVIEW

File: modular_skyrat/modules/ashwalkers/code/item/clothing/shoes/shoes.dm [hunk 1]
```diff
@@ -20,3 +20,21 @@
...
+	desc = "An ornate set of leg wraps commonly worn by a shaman or chieftain."
+	icon_state = "tribalcuffs"
+	mutant_variants = STYLE_DIGITIGRADE
+
+/obj/item/clothing/shoes/wraps/ashwalker/mundanewraps
+	icon = 'modular_skyrat/modules/ashwalkers/icons/obj/clothing/shoes.dmi'
+	worn_icon = 'modular_skyrat/modules/ashwalkers/icons/mob/clothing/feet.dmi'
+	worn_icon_digi = 'modular_skyrat/modules/ashwalkers/icons/mob/clothing/feet_digi.dmi'
+	name = "tribal leg wraps"
+	desc = "a mundane set of leg wraps often worn by tribal villagers."
```
Comment (tf-4) [inline modular_skyrat/modules/ashwalkers/code/item/clothing/shoes/shoes.dm]:
```suggestion
	desc = "A mundane set of leg wraps often worn by tribal villagers."
```

File: modular_skyrat/modules/ashwalkers/code/item/clothing/under/costumes.dm [hunk 2]
```diff
@@ -47,3 +47,24 @@
...
+	worn_icon = 'modular_skyrat/modules/ashwalkers/icons/mob/clothing/uniform.dmi'
+	name = "loincloth and chestwrap"
+	desc = "A poorly sewn dress made of white materials."
+	icon_state = "chestwrap"
+
+/obj/item/clothing/under/costume/gladiator/ash_walker/caesar_clothes
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
+	worn_icon = 'modular_skyrat/modules/ashwalkers/icons/mob/clothing/uniform.dmi'
+	name = "ash walker tunic"
+	desc = "A tattered red tunic of reddened fabric"
```
Comment (tf-4) [inline modular_skyrat/modules/ashwalkers/code/item/clothing/under/costumes.dm]:
```suggestion
	desc = "A tattered red tunic of reddened fabric."
```

**Comment by tf-4** [comment]
Also put screenshots of the sprites in-game in the PR body pls

**Comment by Detrender** [comment]
> Also put screenshots of the sprites in-game in the PR body pls

done, sorry, new to this.

**Review by GoldenAlpharex — APPROVED** [review]
Code checks out, however, I don't know if Ash Walkers currently have any support for loadouts, so I don't know if you'll actually be able to get your item as one.

**Comment by Detrender** [comment]
> Code checks out, however, I don't know if Ash Walkers currently have any support for loadouts, so I don't know if you'll actually be able to get your item as one.

I can just ahelp with the name of the item asking for it, which I know offhand. I'm used to doing that on other servers.
and no, unfortunately they don't have any support for loadouts.

**Comment by GoldenAlpharex** [comment]
> > Code checks out, however, I don't know if Ash Walkers currently have any support for loadouts, so I don't know if you'll actually be able to get your item as one.
> 
> I can just ahelp with the name of the item asking for it, which I know offhand. I'm used to doing that on other servers. and no, unfortunately they don't have any support for loadouts.

Gotcha, we're good to go then.

**Pull request merged**